core/src/com/unciv/logic/trade/TradeEvaluation.kt

```diff
@@ -274,4 +276,7 @@ class TradeEvaluation {
         }
     }
 
+    private fun introductionValue(ruleSet: Ruleset): Int {
+        return ruleSet.modOptions.uniqueObjects.firstOrNull{ it.placeholderText == ModOptionsConstants.tradeCivIntroductions }?.let{ it.params[0].toInt() } ?: 0
```

Comment:
I don't particularly like how long this line is and how many times the 'null' is passed on to the next statement, but other than first checking existence I don't see a simple alternative

Reply:
Concur - split this into val unique = ...firstornull, then later checks